pkg/chartutil/create.go

```diff
@@ -93,6 +93,10 @@ nodeSelector: {}
 tolerations: []
 
 affinity: {}
+
+# Override chart app names with caution, here be dragons!
```

Comment:
I'd probably remove that comment. Could be more confusing than helpful. Plus, I'd put those at the very top of the file.